src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2635,6 +2626,26 @@ Estimated\ time\ left\:\ more\ than\ 2\ minutes.=Estimated time left: more than
 You\ find\ information\ about\ the\ privacy\ policy\ here.=You find information about the privacy policy here.
 The\ size\ of\ the\ embedding\ model\ could\ be\ smaller\ than\ written\ in\ the\ list.=The size of the embedding model could be smaller than written in the list.
 Embedding\ model\ has\ to\ be\ provided=Embedding model has to be provided
+After\ the\ file\ will\ be\ ingested,\ you\ will\ be\ able\ to\ chat\ with\ it=After the file will be ingested, you will be able to chat with it
+Could\ not\ find\ path\ for\ a\ linked\ file\ '%0'\ while\ generating\ embeddings=Could not find path for a linked file '%0' while generating embeddings
+File\ %0\ could\ not\ be\ ingested=File %0 could not be ingested
+File\ %0\ is\ currently\ being\ processed=File %0 is currently being processed
+File\ %0\ is\ not\ a\ PDF\ file=File %0 is not a PDF file
+Generating\ embeddings\ for\ file\ '%0'=Generating embeddings for file '%0'
+No\ citation\ key=No citation key
+No\ files\ attached=No files attached
```

Comment:
Didn't we have a similar message before? 

Can't we resue the following string?

![image](https://github.com/user-attachments/assets/a6188e6a-943d-4c69-88d4-79d4678a77ae)

Reply:
Renamed notifications, so probably this is outdated comment